brainstormforce/ultimate-addons-for-gutenberg
# Improvement: Added compatibility with WordPress v5.7.

### Description

Changelog V1.21.1
* Improvement: Added compatibility with WordPress v5.7.
* Fix: Post Layouts - Fixed the JS undefined error with `post_type` in block editor while using Post Masonry.
* Fix: Post Layouts - Fixed the Post Layout keeps loading when clicking on categories All.

### Screenshots


### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Check UAG Dashboard when WP5.7 is activated.[Working fine]
- Activate/Deactivate block.[Working fine]
- Drag and drop every block.[Working fine]
- Update and refresh the page with no validation issue when uag block on a page.
- Install Migrate helper plugin and check is there a jquery migrate error or not
- Drag and drop post block and check every setting for post and page is working fine or not. (like pagination for all post, order by, order, categories n all)

### Checklist:
